src/frontend/web_application/src/services/openpgp-keychain-repository/local-storage-helper.js

```diff
@@ -0,0 +1,20 @@
+export function findAll(namespace) {
+  return Object.keys(window.localStorage)
```

Comment:
added a test in top of file. The following code is only be present on browser built